api/apis/space_handler.go

```diff
@@ -88,6 +88,13 @@ func (h *SpaceHandler) SpaceCreateHandler(info authorization.Info, w http.Respon
 			return
 		}
 
+		if repositories.IsPermissionDeniedOrNotFoundError(err) {
```

Comment:
We created a ForbiddenError as part of the last story. Might be controversial, but you could use that here and in the repo.

Reply:
Thanks! Using the `ForbiddenError` does make sense here as it is a bit weird to get a "bla-bla-or-does-not-exist` error when you are creating new stuff.